RNA-seq/02-gastric_cancer_tximeta.Rmd

```diff
@@ -0,0 +1,174 @@
+---
+title: "Gastric cancer: gene-level summarization with `tximport`"
```

Comment:
I am wondering if we should include `tximeta` in the title as well, I felt strongly that we should at first glance, to stay consistent with the file name, but now I feel less strongly.

Reply:
Yeah I haven't been 100% sure about when to reference `tximeta` and when to reference `tximport`.